.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: Create release candidate
+
+on: [workflow_dispatch]
+
+jobs:
+  create_release:
+    name: Create release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/cache@v3
+        with:
+          path: ./node_modules
+          key: npm-cache-${{ runner.os }}-${{ hashFiles('./package-lock.json') }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Run release script and open PR
+        run: |
+          git config user.name "API Team CI User"
+          git config user.email "aciattestteamci@deque.com"
+
+          Branch="release-$(date +%Y-%m-%d)"
+          echo "Branch: $Branch"
+          git checkout -b "$Branch"
+
+          npm ci
+          npm run release
+
+          git push origin "$Branch" --force
```

Comment:
Why the force push?

Reply:
If the branch already has changes in it, we'll want to overwrite them.

Reply:
Wouldn't the checkout action fail then and kill the build before it got to this point?

Reply:
No, the checkout action is taking whatever branch we're releasing from (usually `develop`). We then create this branch, add the release commit, then push. The `--force` is to ensure any prior pending release (maybe someone made a mistake) is overwritten.

This is, however, fixing a problem we haven't had (yet?). I'm happy to pull it out if you aren't comfortable force pushing.

Reply:
Wilco didn't seem to think it was a problem when he reviewed it, so I'm fine with it.